### ISSUE TITLE
The subscription '9b445647-9856-42d5-9482-24eeaa3f9e6e' is not allowed to create or update the serverfarm.

### ISSUE THREAD

**Issue by durmusgulbahar** [microsoft/vscode-azureappservice#2631]
Does this occur consistently? 
Repro steps:

1.
2.

Version: 0.25.2
OS: win32
OS Release: 10.0.19045
Product: Visual Studio Code
Product Version: 1.88.1
Language: en

**Comment by durmusgulbahar** [comment]
I followed this instructions : 
https://learn.microsoft.com/en-us/azure/app-service/quickstart-nodejs?tabs=windows&pivots=development-environment-vscode

**Comment by motm32** [comment]
Following the instructions you linked are you able to successfully create a Node.js web app using the portal or CLI?

**Comment by durmusgulbahar** [comment]
nope, I just deployed a web app - database once but I do not understand why it created. I tried everything to create a web app, from vscode from azure portal. Changed regions, I tried germany west, east us, central us. I tried basic, premium plans. But no everytime it returns same error. I really do not understand what is happening. I waited for a day because there would be a problem on servers but no. I tried different internet connections but nope.

**Comment by motm32** [comment]
Thanks for your reply. In this case your best option would be to contact Azure support. You should be able to do this from within the portal by clicking the question mark in the top right hand corner: 
![image](https://github.com/microsoft/vscode-azureappservice/assets/59709511/ef7435a0-7ffd-4d8e-922b-1f0b5ca79dc2)

Hope that helps!

**Comment by svenikea** [comment]
Nope I have the same issue with @durmusgulbahar. Tried everything including different region, unregister and register `Microsoft.Web` no result whatsoever. Tried the support on Azure Portal, it its useless, they greyed out the `next` button at the end so you can't create a support ticket, they literally tell you change your support plan or ask for support on the community forum

**Comment by alexweininger** [comment]
Thanks for reporting this issue. It looks like the Azure App Service is experiencing some service side issues. Creating or updating web apps is intermittently failing. Please try again later, and if we get updates we'll let you know.

**Comment by alexweininger** [comment]
Closing as a duplicate of #2635
